assets/queries/openAPI/schema_discriminator_property_not_string/query.rego

```diff
@@ -109,6 +110,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	schema := doc.components.responses[r].content[c].schema
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)